### ISSUE TITLE
Cross platform config sync

### ISSUE THREAD

**Issue by viharm** [hluk/CopyQ#1141]
**Is your feature request related to a problem? Please describe.**
I use CopyQ on several desktops - two are Windows, and one is Linux. I  use third party sync tools (currently Seafile, previously Syncthing) to synchronise the "config" folder of portable installations. This allows me to synchronise both the tab data as well as configuration (I tried the synchronisation plugin, but the installation paths on all my desktops are different, so it does not work; and it does not sync configuration). This setup works well on my two Windows desktops (I have disabled sync of my clipboard tab using sync filters in my sync utility). Unfortunately this is not viable on my Linux desktop because the configuration file name is different (copyq.ini on Windows & copyq.conf on Linux)

**Describe the solution you'd like**
The structure of the configuration directory is similar in both Windows (C:\PortableApps\CopyQ\config\copyq\) and Linux (~/.config/copy/). It would be ideal to have identical file names so any changes in configuration as synchronised across all instances.

**Describe alternatives you've considered**
I have tried forcing a sync of the above directories, but with different configuration files, I still have to manually mirror changes.

The only other alternative is to install a separate utility to synchronise both files (copyq.ini & copyq.conf) to ensure changes are synced, but this is neither efficient nor graceful.

**Additional context**
Although built-in sync of configuration would be ideal, this is not required as part of this feature request. Only making the config directory identical is requested, including the configuration file itself.

**Comment by hluk** [comment]
Yeah, this is really annoying, I should try to migrate the configuration to `ini` file.

There was a task some time ago to move configuration from window registry to the file, unfortunately, I noticed too late that the the file suffix is different.

**Comment by hluk** [comment]
Maybe as workaround you can create symlink on Linux.

**Comment by viharm** [comment]
The short term fix sounds logical, thanks. I will try it and provide feedback.

As for the long term fix, do you believe this is easily do-able? How long will it take to commonise the file name?

**Comment by viharm** [comment]
Oh by the way, _CopyQ_ has changed my life. I have used many clipboard tools for more than a decade, but nothing as powerful as _CopyQ_. The fact that it's open-source and cross-platform is the icing on the cake. I can't even begin to describe how awesome it is to receive a response from the developer (you) within 24 hours.

Thank you very much for your hard work with _CopyQ_.

**Comment by hluk** [comment]
> As for the long term fix, do you believe this is easily do-able? How long will it take to commonise the file name?

Probably not very difficult to implement, but it can break compatibility with previous versions in some unexpected ways.

If there is an easy workaround, this issue has low priority.

> Oh by the way, CopyQ has changed my life.

Great to hear. Thanks for using CopyQ! :)

**Comment by dmsherazi** [comment]
This will be a great feature! +1
